clustermesh/clustermesh.go

```diff
@@ -140,9 +140,9 @@ func (k *K8sClusterMesh) generateService() (*corev1.Service, error) {
 			svc.Spec.Type = corev1.ServiceTypeLoadBalancer
 			svc.ObjectMeta.Annotations["service.beta.kubernetes.io/azure-load-balancer-internal"] = "true"
 		case k8s.KindEKS:
-			k.Log("🔮 Auto-exposing service within AWS VPC (service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0")
+			k.Log("🔮 Auto-exposing service within AWS VPC (service.beta.kubernetes.io/aws-load-balancer-scheme: internet-facing")
 			svc.Spec.Type = corev1.ServiceTypeLoadBalancer
-			svc.ObjectMeta.Annotations["service.beta.kubernetes.io/aws-load-balancer-internal"] = "0.0.0.0/0"
+			svc.ObjectMeta.Annotations["service.beta.kubernetes.io/aws-load-balancer-scheme"] = "internet-facing"
```

Comment:
Looking at the current annotation, I'm actually puzzled about the intended behavior, given that `service.beta.kubernetes.io/aws-load-balancer-internal` is expected to take a boolean parameter, while we are setting `0.0.0.0/0`. Yet, considering the log message above and the configurations for the other cloud providers, I assume that the goal would be to expose it internally only. For that reason, I'd suggest setting the load balancer scheme to `internal`.

Reply:
The assumption is that the two clusters are either on the same VPC, or they have been peered together, to enable cross-cluster node to node and pod to pod communication.